Feature/feature register

SNKz0610 opened the pull request:
2 changes:
- Convert file MainActivity from Java to Kotlin
- In file activity_main.xml, I change ImageButton of 4 functions under (continue with...) into Button with image and text to make sure that the button can be changed the language of the text. Next, I change background of those buttons into transparency

SNKz0610 committed:
chore(): task #1005
Adding Android homework 10/05/2021 about first month ending by using git flow.Closes task #1005

SNKz0610 committed:
chore(): task#1105
Convert class MainActivity from Java to Kotlin.
Fixing buttons, incluing change ImageButto to Button at 4 functions of continue with.... and change the background into transparency
Closes #task1105

linhvnguyen9 commented:
B tạo pull request mới cũng đc, nhưng nó hơi thừa :)))). Khi b push thay đổi mới lên GitHub nó sẽ tự cập nhật cái pull request cũ :)))

linhvnguyen9 reviewed (approved):
B sửa lại nhìn ok hơn hẳn r đó :+1: :+1: :+1:

linhvnguyen9 commented:
b có thể accept cái pull request này để merge nhánh vào develop và code tiếp part 2 nhé!

SNKz0610 committed:
chore(): task#1105
Adding function for EditText email and Button continue. I mean when app running, user cant click continue button and its color is grey. When they typing something in the email edittext, the button will become clickable and turn into re color.
Closes task#1105

linhvnguyen9 reviewed (approved):
ngon nghẻ r đó bro!

JKhanh commented:
Nếu người dùng nhập vào ký ựt xong r xóa đi thì nút continue của b có cho click không ?

SNKz0610 committed:
chore(): task#1205
Adding part 3 of excercise: set function checking format of email for button Continue, adding relationship between edittext Email and button Continue.
Closes task#1205
